lib/util.js

```diff
@@ -16,3 +16,21 @@ exports.normalizeRoutePrefix = function (prefix) {
     let routePrefix = prefix.trim();
     return routePrefix;
 };
+
+exports.STATSD_METHODS = [
+    'counter',
+    'timing',
+    'gauge',
+    'set',
+    'close',
+    'connect',
+];
+
+exports.isCustomClient = function (obj) {
```

Comment:
I don't understand this function, you can not pass a client with internal variables!

Reply:
I am not sure what you mean by that. I moved the custom client option in the `client` config key. The check is just to see if the object/function/class passed has one of the client public methods.

Reply:
but if for some reason the client has some parameter that is not a function, this may return false!

Reply:
🤔 Ok I think I see the mistake I made here, I was sure I was checking the `STATSD_METHODS` instead I am just checking random properties. I declared the array of methods but forgot to use it. Is that what you meant?

Reply:
yes :+1:

Reply:
Ok, thanks 😉